### ISSUE TITLE
Where is the widget?

### ISSUE THREAD

**Issue by fardina** [arjunsuresh/q2a-quotes#1]
Hi
thanks for the plugin.
Where is the widget?
There is just a quote input form that I added some quotes.
Where can I set where the widget shows the quotes?

**Comment by arjunsuresh** [comment]
oh. You are welcome. The plugin is not complete yet. The widget part is not added as I use this for a custom Home page.

**Comment by fardina** [comment]
Could you please tell me if this plugin will be completed soon or not?

**Comment by arjunsuresh** [comment]
You can try the latest commit - it is not tested though.

**Comment by fardina** [comment]
Thnak you.
But it's not possible to enable event_scheduler in shared hosting! :(
Why do you need event_scheduler to be enabled?
Maybe it's possible to use Cron-job instead of enabling event_scheduler, isn't it?

**Comment by arjunsuresh** [comment]
yes, event scheduler is required to change the quote at midnight everyday. This can be done via cron also - but avoiding cron is better for many users due to ease of server migration, copying etc. May be you can also use a new Quote plugin released on Q2A which works via JS and there is no need to add custom quotes.

**Comment by fardina** [comment]
I need to add quotes because of my language.
I was wondering if you could make plugin work with Cron?
